week1_session4_intro_to_ml_and_scikit_learn.ipynb

```diff
@@ -0,0 +1,1547 @@
+{
```

Comment:
<p>Maybe show one example, where they can see an assertion error? Just for fun :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/ai_in_medicine/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>also a good idea :) done</p>